src/components/FieldLevelHelp/FieldLevelHelp.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon } from '../Icon';
+import { Popover } from '../Popover';
+import { Tooltip } from '../Tooltip';
+import { OverlayTrigger } from '../OverlayTrigger';
+
+/**
+ * FieldLevelHelp Component for Patternfly React
+ */
+const FieldLevelHelp = ({ children, contentType, content, ...props }) => {
+  const trigger = contentType === 'popover' ? 'click' : 'hover focus';
+  const htmlContent = (
+    <div
+      dangerouslySetInnerHTML={{
+        __html: content
+      }}
+    />
+  );
+  const overlay =
+    contentType === 'popover' ? (
+      <Popover id="{contentType}">{htmlContent}</Popover>
+    ) : (
+      <Tooltip id="{contentType}">{htmlContent}</Tooltip>
+    );
+  const rootClose = true;
+
+  return (
+    <div>
+      <label>{children + ' '}</label>
```

Comment:
nitpick: please write as: 
```js
<label>{`${children} `}</label>
```

Reply:
Addressed in 2dd035d6034b21b780d5fc031b497172730c9348